controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -319,6 +319,89 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) (*corev
 		// Else: nothing for eBPF at the moment
 	}
 
+	outputFields := []api.OutputField{
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+		},
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "numFlowLogs",
+			Operation: "count",
+		},
+		{
+			Name:      "TimeFlowStartMs",
+			Operation: "min",
+		},
+		{
+			Name:      "TimeFlowEndMs",
+			Operation: "max",
+		},
+		{
+			Name:      "FlowDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "IfDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "AgentIP",
+			Operation: "first",
+		},
+	}
+
+	if helper.IsTCPDropEnabled(b.desired) {
+		outputTCPDropFields := []api.OutputField{
+			{
+				Name:      "TCPDropBytes",
+				Operation: "sum",
+			},
+			{
+				Name:      "TCPDropBytes",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+			{
+				Name:      "TCPDropPackets",
+				Operation: "sum",
+			},
+			{
+				Name:      "TCPDropPackets",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+		}
```

Comment:
Should we manage `TcpDropLatestState` & `TcpDropLatestDropCause` here ?

Reply:
Not sure can explain how that would help from ui I can add them

Reply:
On `FlowCollector` spec side, you can define to log flow / conversations or both.

On frontend side, the query option dropdown show what's available.

In case you are using only conversations, it's usefull to be able to query metrics based on conversations.
These metrics are used for both `Overview` and `Topology` tabs.